app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -229,7 +229,7 @@ class SparkRestClient(sparkConf: SparkConf) {
       }
     }
   }
-  
+
   private def getStageDatas(attemptTarget: WebTarget): Seq[StageDataImpl] = {
```

Comment:
There are other methods(getSparkConfigs, getStageDatas, getExecutorSummaries, getStagesWithFailedTasks) which log messages the same way, need to change that too.

Reply:
Done.